controller/src/main/java/io/pravega/controller/store/stream/ZKStream.java

```
@@ -237,7 +241,32 @@
         return getId().thenCompose(id -> store.getCachedData(creationPath, id, x -> BitConverter.readLong(x, 0))
                 .thenApply(VersionedMetadata::getObject));
     }
-    
+
+    @Override
+    public CompletableFuture<Void> createSubscriber(String subscriber) {
+        return null;
```

Please throw NotImplementedException or UnsupportedException

---

ok. will do